src/test/java/co/omise/live/BaseLiveTest.java

```diff
@@ -6,11 +6,21 @@
     private final String LIVETEST_PKEY = "[YOUR_PKEY]";
     private final String LIVETEST_SKEY = "[YOUR_SKEY]";
     private final String EXPECTED_EMAIL = "[YOUR_EMAIL]";
+    private final String LIVETEST_CHARGE = "[YOUR_CHARGE]";
+    private final String LIVETEST_REFUND = "[YOUR_REFUND]";
 
     String getUserEmail() {
         return EXPECTED_EMAIL;
     }
 
+    String getChargeId() {
+        return LIVETEST_CHARGE;
+    }
+
+    String getRefundId() {
+        return LIVETEST_REFUND;
+    }
+
```

Comment:
I feel these are a bit too specific to one test class for them to be in the `BaseLiveTest` class.

Reply:
Ok, I'll change this.